src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static RegistrationService registrationService;
+    private static final User invalidLoginUser = new User("Jack", "Qwerty123", 19);
+    private static final User invalidPasswordUser = new User("Sviatoslav", "123", 19);
+    private static final User invalidAgeUser = new User("Andrey", "Qwerty321", 17);
+    private static final User nullValueUser = new User("null", "null", 1);
+    private static final User validUser1 = new User("Arcturus", "qwerty654", 18);
+    private static final User validUser2 = new User("Aleksey", "Qwerty", 57);
+    private static final User validUser3 = new User("Oppenheimer", "Qwerty", 25);
+    private static final User emptyUser = new User("", "", 18);
+
+    @BeforeAll
+    static void run() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void clean_storage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void invalid_login_test() {
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(invalidLoginUser));
+    }
+
+    @Test
+    void null_value_test() {
```

Comment:
Please recheck naming of your test methods.
According to task requirements:
For this task use such convention: ```<methodUnderTest>_<state>_<expectedBehavior>```; For example, if we are testing the method register with a null
user's age the test method name should be register_nullAge_notOk. notOk is because the test expects the register method to throw an exception.